### PR TITLE
CE-3317 Enable RWA only on kirkburn.wikia.com

### PR DESCRIPTION
**Pull request by adamkarminski**
@hakubo @rogatty @kamilkoterba

### CODE REVIEW

File: server/app/routes.js [hunk 1]
```diff
@@ -216,6 +216,14 @@ let routes,
 		},
 		{
 			method: 'GET',
+			path: '/diff/{*}',
```
Comment (adamkarminski) [inline server/app/routes.js]: This one is not related to enabling it only for one wiki, please excuse me. :)

File: front/main/app/components/side-nav-local-navigation-menu-2016.js [hunk 2]
```diff
@@ -26,6 +26,10 @@ export default Ember.Component.extend(
 	{
 		tagName: 'ul',
 		classNames: ['local-nav-menu-2016'],
+		// TODO: Remove to enable RWA globally
+		displayRecentWikiActivity: Ember.computed(function () {
```
Comment (bkoval) [inline front/main/app/components/side-nav-local-navigation-menu-2016.js]: Now if this is a boolean, my feeling is it's name should indicate state, not an action, e.g. "shouldDisplayRecentWikiActivity" :)

**Comment by hakubo** [comment]
:1 to @bkoval comment
otherwise LGTM

File: front/main/app/routes/recent-wiki-activity.js [hunk 3]
```diff
@@ -16,6 +16,19 @@ export default Ember.Route.extend(MetaTagsMixin, {
...
 
 	/**
+	 * Make sure that RWA is enabled only for one wiki.
+	 * @param {EmberState.Transition} transition
+	 * @return {void}
+     */
+	beforeModel(transition) {
+		// Enable the RWA for kirkburn.wikia.com only
+		if (Ember.get(Mercury, 'wiki.id') !== 41727) {
+			transition.abort();
```
Comment (hakubo) [inline front/main/app/routes/recent-wiki-activity.js]: I don't think you need to call abort here.
I am pretty sure that transitionTo is enough

File: front/main/app/components/side-nav-local-navigation-menu-2016.js (hunk 2 above)
Comment (adamkarminski) [inline front/main/app/components/side-nav-local-navigation-menu-2016.js]: :+1:

**Comment by adamkarminski** [comment]
@bkoval @hakubo - done, thanks!

**Comment by rogatty** [comment]
Would LGTM if linter passed:
```
/var/lib/jenkins/workspace/Mercury-PR-Checks/front/main/app/components/side-nav-local-navigation-menu-2016.js
  30:51  error  Unexpected function expression  prefer-arrow-callback

/var/lib/jenkins/workspace/Mercury-PR-Checks/front/main/app/routes/recent-wiki-activity.js
  23:14  error  "transition" is defined but never used  no-unused-vars

✖ 2 problems (2 errors, 0 warnings)
```

**Comment by adamkarminski** [comment]
Linting passed, merging!
